app/example-page/page.tsx

```diff
@@ -0,0 +1,11 @@
+import ExampleComponent from '../../components/ExampleComponent/ExampleComponent';
```

Comment:
This nested structure is annoying for me when co-locating style files, personally i would prefer single component files without unnecessary folders/style files, only case when folders are good is when grouping components, so the path would look like `@/components/error-message` or `@/components/ErrorMessage`, when grouped `@/components/auth/LoginForm` etc

Reply:
Ah. I did it like this, because VS Code did it for me XD We can add statement in linter for that